dali/base/dautils.cpp

```diff
@@ -3245,9 +3245,25 @@ class CLocalOrDistributedFile: implements ILocalOrDistributedFile, public CInter
                 if (dfile.get())
                     return true;
             }
+
+            StringBuffer dir;
+            StringBuffer cluster;
+#ifdef _CONTAINERIZED
+            // NB: this may work equally as well in bare-metal
+            if (clusters)
+            {
+                if (clusters->ordinality()>1)
+                    throw makeStringExceptionV(0, "Container mode does not yet support output to multiple clusters while writing file %s)", fname);
+                cluster.append(clusters->item(0));
+            }
+            else
+                getDefaultStoragePlane(cluster);
```

Comment:
Not directly related, but this function implementation looks wrong.
Shouldn't it be using @ storagePlane

    const char * plane = config.queryProp("@storagePlane");

Reply:
agreed looks wrong. As existing separate issued, I've opened a separate JIRA : https://track.hpccsystems.com/browse/HPCC-25903